azext_iot/tests/iothub/core/test_iot_ext_int.py

```diff
@@ -7,15 +7,18 @@
 import os
 import pytest
 from time import sleep
+from knack.util import CLIError
```

Comment:
The module `test_iot_ext_int.py` has undergone a lot of changes and removals of prior tests in the last few months, we should change the name to something that better reflects its test focus.